cssom-view-1/Overview.bs

```diff
@@ -413,16 +414,24 @@ protect the user's privacy. In order to do so in a consistent manner across APIs
 defines the following terms, each having a width and a height, the origin being the top left
 corner, and the x- and y-coordinates increase rightwards and downwards, respectively.
 
-The <dfn export>Web-exposed screen area</dfn> is one of the following:
+The <dfn export>Web-exposed screen area</dfn> must return the result of the following algorithm:
 
-* The area of the output device, in <a lt=px value>CSS pixels</a>.
-* The area of the <a>viewport</a>, in <a lt=px value>CSS pixels</a>.
+1. Let <var>target</var> be [=this=]'s [=relevant global object=]'s [=Window/browsing context=].
+1. Let |emulated screen area| be the <a>WebDriver BiDi emulated screen area</a> for <var>target</var>.
```

Comment:
Let's split this into "WebDriver BiDi emulated total screen area" and "WebDriver BiDi emulated available screen area".